### PR TITLE
adding docs for config files

### PR DESCRIPTION
**Pull request by andy-berry-dev**
adding a page at '/docs/use/config/' for docs on the various config files.

### CODE REVIEW

File: _includes/docs/use/toc.html [hunk 1]
```diff
@@ -4,4 +4,5 @@
   <li><a href="/docs/use/commandline/">Using the brjs CLI</a></li>
   <li><a href="/docs/use/writing_tests/">Writing Tests</a></li>
   <li><a href="/docs/use/running_tests/">Running Tests</a></li>
+  <li><a href="/docs/use/config/">Config Options</a></li>
```
Comment (leggetter) [inline _includes/docs/use/toc.html]: Could this be a bit ambiguous? Your pull request name is `Conf File`. So maybe something along those lines is more descriptive?

Comment (andy-berry-dev) [inline _includes/docs/use/toc.html]: 'Config Files'...?

Comment (leggetter) [inline _includes/docs/use/toc.html]: :+1:

**Comment by andy-berry-dev** [comment]
@leggetter TOC now says 'Config Files'

**Pull request merged**